spec/requests/content_item_requests/downstream_requests_spec.rb

```diff
@@ -19,23 +20,13 @@
     let(:request_method) { :put }
 
     sends_to_draft_content_store
-
-    it "sends to live content store" do
-      expect(PublishingAPI.service(:live_content_store)).to receive(:put_content_item)
-        .with(
-          base_path: base_path,
-          content_item: content_item_for_live_content_store,
-        )
-        .and_return(json_response)
-        .ordered
```

Comment:
We're calling `sends_to_live_content_store` further down. There was already a shared support method for this.